### PR TITLE
Prosthetics Additions 2

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds 5 new prosthetics for the prosthetics clinic:

Fixer tools - A tool implant that holds 4 tools, a PDA, a flashlight, a combat knife and a crowbar.
It's cheap! But requires surgery.

Timestop arms - A sidegrade to the explosive arms that drops a timestop instead of killing you.

Briefcase Implant - An implant that holds a briefcase for storage

Overdrive - an implant that deals toxin damage on activation; but increases movement speed by 50% and retracts after 10 seconds.

Observation - a Scrying Orb implant. Extremely expensive

## Why It's Good For The Game
More Prosthetics are always great!

## Changelog
:cl:
add: Added 5 new prosthetics
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_items/prosthetics/scrying.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/obj/item/organ/cyberimp/arm/observation
+	name = "obvservation implant"
+	desc = "A device that lets you oversee the surrounding area."
+	icon = 'ModularTegustation/tegu_items/prosthetics/icons/generic.dmi'
+	icon_state = "investigation"
+	contents = newlist(/obj/item/scrying/observation)
```
Comment (InsightfulParasite) [inline ModularTegustation/tegu_items/prosthetics/scrying.dm:6]: You sure you want to give them a actual scrying orb?

Comment (InsightfulParasite) [reply ModularTegustation/tegu_items/prosthetics/scrying.dm:6]: Scrying orbs for wizards can let the user ghost if they use it in hand.
They also allow the user to hear deadchat.

**Comment by Kitsunemitsu** [comment]
I will be using the timestop arms as special arms in ruins.

**Review by InsightfulParasite — APPROVED** [review]
Observation seems a bit powerful but ill let gameplay show how good it is. Approval'd

**Pull request merged**